src/browser-extension/popup/toggle/toggle.tsx

```diff
@@ -0,0 +1,20 @@
+import React from 'react';
+
+import './toggle.scss';
+
+declare const chrome: any;
+
+export default class Toggle extends React.Component<{}, {}> {
+  constructor(props: {}) {
+    super(props);
```

Comment:
I think it's necessary for this component to contain text that is clickable and allows to activate/deactivate the toggle button.
I'll fix it soon.